package.json

```diff
@@ -8,11 +8,10 @@
     "url": "git://github.com/menpo/landmarker.git"
   },
   "scripts": {
-    "build": "DISABLE_NOTIFIER=true NODE_ENV=production gulp build",
-    "watch": "gulp",
+    "build": "NODE_ENV=production ./node_modules/gulp/bin/gulp.js build",
+    "watch": "./node_modules/gulp/bin/gulp.js",
```

Comment:
No need in the npm scripts, when you run `npm run build`, the PATH has the local binaries